cypress/integration/host/host_events.ts

```diff
@@ -0,0 +1,125 @@
+const pathWithEvents = `/host/i-0f81a2d39744003dd`;
+
+describe("Host events", () => {
+  before(() => {
+    cy.login();
+  });
+
+  beforeEach(() => {
+    cy.listenGQL();
+    cy.preserveCookies();
+  });
+
+  it("host events display the correct text", () => {
+    cy.visit(pathWithEvents);
+    cy.dataCy("created").contains("Host created");
+    cy.dataCy("agent-deploy-failed").contains("New agent deploy failed");
+    cy.dataCy("provision-error").contains(
+      "Host encountered error during provisioning"
+    );
+    cy.dataCy("started").contains("Host start attempt succeeded");
+    cy.dataCy("stopped").contains("Host stop attempt");
+    cy.dataCy("modified").contains("Host modify attempt");
+    cy.dataCy("agent-deployed").contains(
+      "Agent deployed with revision 2019-05-25 from f4c4c42abc123456d14edfe4c123bb1a1a47dd12"
+    );
+    cy.dataCy("agent-monitor-deployed").contains(
+      "Agent monitor deployed with revision 1fa212ac4acea6ce2a0123a123456c06e8cf72ea"
+    );
+    cy.dataCy("agent-monitor-deploy-failed").contains(
+      "New agent monitor deploy failed"
+    );
+    cy.dataCy("host-jasper-restarting").contains(
+      "Jasper service marked as restarting by mci"
+    );
+    cy.dataCy("host-jasper-restarted").contains(
+      "Jasper service restarted with revision"
+    );
+    cy.dataCy("host-converting-provisioning").contains(
+      "Host converting provisioning type"
+    );
+    cy.dataCy("host-converted-provisioning").contains(
+      "Host successfully converted provisioning type"
+    );
+    cy.dataCy("host-dns-name-set").contains(
+      "DNS Name set to ec2-12-12-12-122.compute-1.amazonaws.com"
+    );
+
+    cy.dataCy("host-provisioned").contains("Marked as provisioned");
+
+    cy.dataCy("host-task-pid-set").contains("PID of running task set to 123");
+    cy.dataCy("host-monitor-flag").contains(
+      "Flagged for termination because: unreachable"
+    );
+
+    cy.dataCy("host-expiration-warning-set").contains(
+      "Expiration warning sent"
+    );
+    cy.dataCy("host-running-task-set").contains(
+      "Assigned to run task evergreen_ubuntu1604_test_command_patch_5e823e1f28..."
+    );
+    cy.dataCy("host-running-task-cleared").contains(
+      "Current running task cleared (was:evergreen_ubuntu1604_test_command_patch_5e823e1f28...)"
+    );
+    cy.dataCy("host-task-finished").contains(
+      "Task evergreen_ubuntu1604_test_command_patch_5e823e1f28... completed with status: test-timed-out"
+    );
```

Comment:
rather than asserting the correct data line by line, which is difficult to read and unlikely maintainable, you can generate tests in a loop. take a look at the hosts table tests for examples of this.